home work 21/script.js

```diff
@@ -0,0 +1,67 @@
+ul.addEventListener("click", (event) => {
+  const target = event.target;
+  if (!target.matches(".list_item")) return;
+
+  if (event.ctrlKey) {
+    target.classList.toggle("highlighted");
+    lastHighlighted = target;
+  } else if (event.shiftKey) {
+    const start = Array.from(ul.children).indexOf(
+      lastHighlighted || ul.children[0]
+    );
+    const end = Array.from(ul.children).indexOf(target);
+    Array.from(ul.children)
+      .slice(Math.min(start, end), Math.max(start, end) + 1)
+      .forEach((li) => li.classList.add("highlighted"));
+    lastHighlighted = target;
+  } else {
+    ul.querySelectorAll(".highlighted").forEach((li) =>
+      li.classList.remove("highlighted")
+    );
+    target.classList.add("highlighted");
+    lastHighlighted = target;
+  }
+});
+
+const addTopBtn = document.querySelector(".btn-add-beginning");
+const addBottomBtn = document.querySelector(".btn-add-end");
+const deleteBtn = document.querySelector(".btn-delete");
+const sortBtn = document.querySelector(".btn-sort");
+const input = document.querySelector(".input");
+
+addTopBtn.addEventListener("click", () => {
+  const li = document.createElement("li");
+  if (input.value === "") return;
+  li.classList.add("list_item");
+  li.textContent = input.value;
+  ul.prepend(li);
+  input.value = "";
+});
+
+addBottomBtn.addEventListener("click", () => {
+  const li = document.createElement("li");
+  if (input.value === "") return;
+  li.classList.add("list_item");
+  li.textContent = input.value;
+  ul.appendChild(li);
+  input.value = "";
+});
```

Comment:
wrong logic with addBottomBtn, you need to add li element with random text on bottom of the list